techmatters/hrm
# CHI-2296 ancestor taxonomies

## Description

* Tweaks the import mappings to import the ancestor taxonomies that are now provided 
* In our flattened structure we don't differentiate between leaf taxonomies & branch taxonomies, they just get added as taxonomy attributes, because we only use them as search index fodder - it's easy to infer the hierarchy from the codes should we ever need to - each child extends the code of its parent, e.g. T -> TJ -> TJ-3000
* Updated the reindex script to assume a role before attempting to do its work

### Checklist

### Related Issues
CHI-2296

### Verification steps

Reimported & tested changes on dev - they look to be working, ancestor taxonomies are being used in search results. A lot of the source data is missing the ancestor taxonomies but that's for Arctic to fix, if we can see the change working on the resources where the data is present I think we can go ahead with it
